js-callback/js-callback-functions_user-login-callback-3-with-multiple-arguments/js/index.js

```diff
@@ -5,3 +5,18 @@ function handleUserLogin(onSuccess) {
 }
 
 // The exercise starts here!
+//1. With additional Callback
+
+const showWelcomeMessage = (userName, userRole) => {
+  console.log(`Welcome ${userName}! You are logged in now as ${userRole}.`);
+};
+
+handleUserLogin(showWelcomeMessage);
+
+//2. Within Function Call
+
+handleUserLogin((userName, userRole) => {
+  console.log(
+    `Welcome Again! ${userName}! You are logged in now as ${userRole}.`
+  );
```

Comment:
Or you can simply do
```suggestion
  showWelcomeMessage(userName, userRole);
```